schemas/event-schema.json

```diff
@@ -0,0 +1,77 @@
+{
+	"$schema": "http://json-schema.org/draft-07/schema#",
+	"$id": "http://schema.esamarathon.com/mq/event-schema",
+	"title": "European Speedrunner Assembly RabbitMQ event schema",
+	"definitions": {
+		"nonEmptyString": {
+			"type": "string",
+			"minLength": 1
+		},
+		"timestampObject": {
+			"type": "object",
+			"required": [
+				"iso",
+				"unix"
+			],
+			"properties": {
+				"iso": {
+					"type": "string",
+					"description": "Timestamp representation in the ISO 8601 format",
+					"format": "date-time",
+					"examples": [
+						"2019-09-03T19:55:18.430Z"
+					]
+				},
+				"unix": {
+					"type": "number",
+					"description": "Timestamp representation in seconds since the Unix epoch, including a fractional millisecond part",
+					"examples": [
+						1567540518.430
+					]
+				}
+			},
+			"additionalProperties": false
+		}
+	},
+	"properties": {
+		"$meta": {
+			"type": "object",
+			"required": [
+				"vHost",
```

Comment:
The vHost can probably be removed entirely. It's a configuration detail of the specific server/application in question, not of the event.